Makefile

```diff
@@ -167,33 +163,32 @@ msi_version := $(VERSION:dev=0.0.1)
 
 # This builds the Windows installer (MSI) using platform-dependent WIX commands.
 dist/func-e_$(VERSION)_%.msi: build/func-e_%/func-e.exe.signed
+ifeq ($(OS),Windows_NT)  # Windows 10 etc use https://wixtoolset.org
 	@printf "$(ansi_format_dark)" msi "building $@"
 	@mkdir -p $(@D)
-ifeq ($(goos),windows)  # Windows 10 etc use https://wixtoolset.org
 	@candle -nologo -arch $(call msi-arch,$@) -dVersion=$(msi_version) -dBin=$(<:.signed=) packaging/msi/func-e.wxs
 	@light -nologo func-e.wixobj -o $@ -spdb
 	@rm func-e.wixobj
```

Comment:
funnily enough, this was already possible for func-e.

Reply:
oops..